angular/package.json

```diff
@@ -10,10 +10,13 @@
     "type": "git",
     "url": "https://github.com/TheIronYard--Orlando/TIY-Catalog"
   },
-  "author": "David Rogers <david@theironyard.com>",
+  "author": "David Rogers <david@theironyard.com>, Taylor Foster <tagfoster90@gmail.com>",
   "license": "GPL",
   "bugs": {
     "url": "https://github.com/TheIronYard--Orlando/TIY-Catalog/issues"
   },
-  "homepage": "https://github.com/TheIronYard--Orlando/TIY-Catalog"
+  "homepage": "https://github.com/TheIronYard--Orlando/TIY-Catalog",
+  "dependencies": {
+    "lodash": "^3.3.1"
+  }
```

Comment:
Kinda really a `"devDependency"` because we don't use `lodash` to _deploy_ our package, just to _maintain_ it.